src/SFA.DAS.EAS.Web.IntegrationTests/HealthCheckControllerTests/WhenHealthCheck.cs

```diff
@@ -17,10 +17,10 @@ public async Task ThenTheStatusShouldBeOk()
                             .AllowStatusCodes(HttpStatusCode.OK);
 
             // Act
-            await ApiIntegrationTester.InvokeIsolatedGetAsync(call);
+            var response = await ApiIntegrationTester.InvokeIsolatedGetAsync(call);
 
             // Assert
-            Assert.Pass("Verified we got OK");
+            Assert.That(response.Response.StatusCode == HttpStatusCode.OK);
```

Comment:
This is minor but I think this assertion only tests that it is true. Using Assert.Equals(…) or the fluent equivalent will show the expected value and the actual value , which would be more useful in the test messages.